dockyard/common/network/network.py

```diff
@@ -0,0 +1,33 @@
+import ast, json
```

Comment:
Find out lot of proposals for this class and volumes class as these are currently being developed.